CMakeLists.txt

```diff
@@ -61,11 +75,17 @@ list(APPEND CUDA_NVCC_FLAGS_DEBUG "-O0 -g")
 if(${CMAKE_VERSION} VERSION_LESS 2.8.10)
     list(APPEND CUDA_NVCC_FLAGS "-ccbin ${CMAKE_C_COMPILER}")
 endif()
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
```

Comment:
Can you add a comment, why this is required? Looks like a work around/hack for a specific version. If possible include link to bug report.